lib/Support/RTL/RTL.cpp

```diff
@@ -294,7 +294,8 @@ void RTLMatch::registerParameters(hw::HWModuleExternOp &modOp) {
   if (name == "handshake.addi" || name == "handshake.buffer" ||
```

Comment:
this file is in the older version. new version in #330 registers each parameter in a dedicated method